### DISCUSSION TITLE
Automatically continue unfinished task

### DISCUSSION THREAD

**Discussion by daniellionel01** [reworkd/AgentGPT#62] [Ideas]
I played around a lot with this project and I experience unfinished tasks quite often. For example it generates a paragraph and then just stops mid sentence. This is a typical behaviour of GPT 3.5 and 4 when the output becomes very long. Usually you can get the model to finish its output by just asking it to "continue".

Since the output of a task is pretty much always ended with a dot "." you could have a very simple if-statement that checks if the output of GPT ends with a dot and if not, prompt it to "continue". There are probably edge cases or other ideas to get it to always finish a task, but it's definitely a big inconvenience in its use.

**Comment by jzuniga-tdmcs** [comment]
In working with GPT-4, I have found that when it fails to complete a response, the following exchange is effective:
user: Please complete your prior response.
ai: I apologize for the incomplete response. Here is the continuation of...

This may be another option for prodding a slow or faulted response.
